recipes/intel-neon2sse/all/conanfile.py

```diff
@@ -13,8 +12,6 @@ class IntelNeon2sseConan(ConanFile):
     license = "BSD-2-Clause"
     topics = "neon", "sse", "port", "translation", "intrinsics"
     settings = "os", "compiler", "build_type", "arch"
```

Comment:
Could you add a `package_type` attribute? https://docs.conan.io/2.0/reference/conanfile/attributes.html#package-type